src/components/Capture/ProfileData.tsx

```diff
@@ -0,0 +1,140 @@
+import { h, Component } from 'preact'
```

Comment:
@zoeradkani promised to completely redo this component, so we skip it in the review.

Reply:
I'll be working on this component, it is changing based on new designs.